README.md

```diff
@@ -61,6 +61,81 @@ error about `setuptools-rust` not being found you should upgrade pip with
 `pip install -U pip` or manually install `setuptools-rust` with
 `pip install setuptools-rust` and try again.
 
+### Platform Support
+
+Retworkx strives to support as many platforms as possible, but due to
+limitations in available testing resources and platform availability, not all
+platforms can be supported. Platform support for retworkx is broken into 4
+tiers with different levels of support for each tier. For platforms outside
+these, retworkx is probably still installable, but it’s not tested and you will
+need a Rust compiler and have to build retworkx (and likely Numpy too) from
+source.
+
+Additionally, retworkx only supports CPython. Running with other Python
+interpreters isn’t currently supported.
+
+#### Tier 1
+
+Tier 1 supported platforms are fully tested upstream as part of the development
+processes to ensure any proposed change will function correctly. Pre-compiled
+binaries are built, tested, and published to PyPI as part of the release
+process. These platforms are expected to be installable with just a functioning
+Python environment.
+
+Tier 1 platforms are currently:
+
+- Linux x86_64 (distributions compatible with the manylinux 2010 packaging
+  specification)
+- macOS x86_64 (10.9 or newer)
+- Windows 64 bit
+
+
+#### Tier 2
+
+Tier 2 platforms are not tested upstream as part of development process.
```

Comment:
It's a bit inconsistent whether you say "the development process" or "the development processes" in each tier - probably best to choose one and stick to it.  This one here in tier 2 is missing a definite article whichever way you go.

(I'm aware that this text is copy/pasted from Qiskit's version - I apparently missed it the first time around!)

Reply:
I settled on the "the development process" in: https://github.com/Qiskit/retworkx/pull/559/commits/e09e02133053b7a84d99d7fdb7fa608c34742ee5